web/themes/custom/pineapple/templates/content/node--recipe--full.html.twig

```diff
@@ -129,4 +129,10 @@
     </aside> <!-- End of recipe-sidebar -->
   </div> <!-- End of row -->
   {{ content.field_r_recipe_comments }}
+  {% set comment_count = node.field_r_recipe_comments.comment_count %}
+  {% if comment_count > 10 and not logged_in %}
+    <div class="forbidden-comments">
```

Comment:
Why forbidden comments?

Reply:
it only shows up when comments are forbidden